data/sql/updates/pending_db_world/rev_1691257289493032818.sql

```diff
@@ -0,0 +1,5 @@
+UPDATE `creature_template` SET `AIName` = 'SmartAI' WHERE `entry` = 17350;
+
+DELETE FROM `smart_scripts` WHERE (source_type = 0 AND entryorguid = 17350);
```

Comment:
don't forget the backticks :)

Reply:
```suggestion
DELETE FROM `smart_scripts` WHERE (`source_type` = 0 AND `entryorguid` = 17350);
```

Other than that I think this looks good =)

Reply:
Thanks, I'll remember that for next time :)